homeassistant/components/apple_tv/strings.json

```
@@ -0,0 +1,68 @@
+        "data": {
+          "identifier": "Device"
+        }
+      },
+      "reconfigure": {
+        "title": "Device Reconfiguration",
+        "description": "This Apple TV is experiencing some connection difficulties and must be reconfigured."
+      },
+      "pair_with_pin": {
+        "title": "Pair With Device",
+        "description": "Pairing is required for the `{protocol}` protocol. Please enter the PIN code displayed on screen. Leading zeros shall be omitted, i.e. enter 123 if the displayed code is 0123.",
+        "data": {
+          "pin": "PIN Code"
+        }
+      },
+      "pair_no_pin": {
+        "title": "Pair With Device",
+        "description": "Pairing is required for the `{protocol}` service. Please enter PIN {pin} on your Apple TV to continue."
+      },
+      "service_problem": {
+        "title": "Failed to Add Service",
+        "description": "A problem occurred while pairing protocol `{protocol}`. It will be ignored."
+      },
+      "confirm": {
+        "title": "Confirm Adding Apple TV",
+        "description": "You are about to add the Apple TV named `{name}` to Home Assistant.\n\n**To complete the process, you may have to enter multiple PIN codes.**\n\nPlease note that you will *not* be able to power off your Apple TV with this integration. Only the media player in Home Assistant will turn off!"
+      },
+      "zeroconf": {
+        "lookup_id_failed": "Failed to look up device id for device."
+      }
+    },
+    "error": {
+      "device_not_found": "Device could not be found on network, please try again.",
+      "device_already_configured": "This device has already been configured. Please pick another device.",
+      "no_usable_service": "A device was found but could not identify any way to establish a connection to it. If you keep seeing this message, try specifying it's IP-address or restarting your Apple TV.",
+      "unknown": "Unexpected error! Check log file for further details.",
+      "auth": "Authentication error (you might have entered an invalid PIN code)."
+    },
+    "abort": {
+      "device_not_found": "Device could not be found on network, please try again.",
+      "already_configured": "Device is already configured!",
+      "updated_configuration": "Updated existing configuration!",
+      "unrecoverable_error": "An unrecoverable occurred! Check log file for further details.",
+      "device_did_not_pair": "No attempt to finish pairing process was made from the device.",
+      "backoff": "Device does not accept pairing reqests at this time (you might have entered an invalid PIN code too many times), try again later.",
+      "invalid_config": "The configuration for this device is incomplete. Please try adding it again.",
+      "auth": "Authentication error. You might have entered an invalid PIN code or not have permission to pair this protocol. Please check access settings on your Apple TV."
```

```suggestion
      "auth": "Authentication error. You might have entered an invalid PIN code or not have permission to pair this Apple TV. Please check access settings on your Apple TV."
```

---

Current text is actually correct since an Apple TV usually supports multiple protocols (MRP and AirPlay for instance) and they are paired independently of each other. So pairing might succeed for MRP but not AirPlay. The nomenclature here might be a bit confusing but I would like to keep the distinction.

---

Thanks for the clarification. What do you think about the following?
```suggestion
      "auth": "Authentication error. You might have entered an invalid PIN code or not have permission to pair via this protocol. Please check access settings on your Apple TV."
```
Emphasizing that the pairing process is carried out over the specific protocol.

---

Sure, that is fine.